src/index.js

```diff
@@ -0,0 +1,46 @@
+import './style.css';
+
+const tasksList = [
+  {
+    description: 'Learn Javscript ES6',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'Practice CSS3 transition',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Learn Webpack',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'First Capstone project',
+    completed: false,
+    index: 3,
+  },
+  {
+    description: 'Learn arrow functions',
+    completed: false,
+    index: 4,
+  },
+];
+
+const showAllTasks = () => {
+  const todoList = document.querySelector('.todo-list');
+  let tasks = '';
+
+  tasksList.forEach((task) => {
+    tasks += `<li class="todo-item">
+    <input type="checkbox" name="check" id="check">
+    <input type="text" name="task" id="task" value="${task.description}">
+    <i class="fa-solid fa-trash-can"></i>
+    </li>`;
+  });
+
+  todoList.innerHTML = tasks;
+};
```

Comment:
- [ ] Kindly sort the list based on their index value to achieve this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.). The list should appear in order of the index values for each task.

Reply:
Thank you for the feedback

Reply:
well resolved!